src/main/java/attendance/AttendanceManagementApplication.java

```diff
@@ -0,0 +1,20 @@
+package attendance;
+
+import java.time.LocalDateTime;
+
+import attendance.controller.AttendanceManagementController;
+import attendance.view.InputView;
+import attendance.view.ResultView;
+
+public class AttendanceManagementApplication {
+
+    public static void main(String[] args) {
+        LocalDateTime today = LocalDateTime.of(2025, 2, 28, 10, 0);
```

Comment:
```suggestion
        LocalDateTime today = LocalDateTime.now();
```
- 원래 처음 생각했던 설계는 항상 오늘을 기준으로 날짜를 받아 출석 관리 시스템을 진행하도록 목표했었습니다.
- 생각보다 Controller를 완성하는 시간이 오래 걸려 해당 부분에 대한 처리를(월이 바뀌어 출석 기록이 없을 때의 실행) 완성하지 못해 2월 28일을 today로 고정했습니다